zuul.d/projects.yaml

```diff
@@ -0,0 +1,33 @@
+- project:
+    check:
+      jobs:
+        - openlab-zuul-jobs-check
+
+- project:
+    name: ansible/ansible
+    third-party-check:
```

Comment:
We can re-use `check`pipeline at here, and if we want to support more OpenStack versions, `recheck-*` pipelines are ready to be used, then we can keep consistent config for ansible, gophercloud and terraform.

Reply:
OK, done